tests/benchmarks/test_benchmark_ddpg.py

```diff
@@ -27,8 +27,8 @@
 import pandas as pd
 import tensorflow as tf
 
+from garage.logger import CsvOutput, logger as garage_logger, TensorBoardOutput
```

Comment:
please don't contract complicated import lines like this
it would be more readable as
```python
from garage.logger import logger  # no need to rename
from garage.logger import CsvOutput, TensorBoardOutput
```

Reply:
ping